packages/pwa/app/utils/polyfills.test.js

```diff
@@ -7,6 +7,7 @@
 import * as polyfills from './polyfills'
 
 test('getNeededPolyfills should return a filtered array of required polyfills', () => {
+    global.fetch = undefined
```

Comment:
if you need to mutate global mock in an individual unit test, you need to make sure that `global.fetch` is re-mocked after this unit test ran.

that being said, I'm surprised other tests didn't break.